.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: false
+language: python
+matrix:
+  include:
+  - python: 2.7
+    env: TOXENV=pep8
+  - python: 2.7
+    env: TOXENV=py27
+  - python: 3.4
+    env: TOXENV=py34
+  - python: 3.5
+    env: TOXENV=py35
+install: pip install tox codecov
+script:
+  - tox
+  - coverage run tests.py
```

Comment:
You can remove the `coverage run tests.py` call as tox will take care of running [this](https://github.com/RackSec/falcon-mutualauth/pull/4/files#diff-b91f3d5bd63fcd17221b267e851608e8R8)